src/Common/src/TypeSystem/CodeGen/MethodDesc.CodeGen.cs

```diff
@@ -50,7 +50,7 @@ public virtual bool IsAggressiveInlining
 
         /// <summary>
         /// Gets a value specifying whether the implementation of this method
-        /// is provided by the runtime.
+        /// is provided by the runtime (i.e., through generated IL).
```

Comment:
Scratch that. That version would be just a documentation of an implementation detail.